tests/keras/engine/test_training.py

```diff
@@ -1173,5 +1173,47 @@ def test_pandas_dataframe():
                           [output_a_df, output_b_df])
 
 
+@pytest.mark.skipif(K.backend() != 'tensorflow',
+                    reason='channels_first with sparse_categorical_crossentropy not yet supported with other backends')
+@keras_test
+def test_model_with_sparse_loss_channels_first():
```

Comment:
About this test:
- please use the same docstring format as elsewhere in the codebase
- use `'` as quote character for consistency
- use lines that are <= 80 char